Makefile

```diff
@@ -1,17 +1,26 @@
+################################################################################
+##                             VERSION PARAMS                                 ##
+################################################################################
+
+## Docker Build Versions
+DOCKER_BUILD_IMAGE = golang:1.13
+DOCKER_BASE_IMAGE = alpine:3.10
+
+## Tool Versions
+TERRAFORM_VERSION=0.11.14
+KOPS_VERSION=1.14.1
+HELM_VERSION=v2.14.2
```

Comment:
we can use helm 2.16.0

Reply:
@gigawhitlocks actually ran into issues using newer helm versions. 2.14.2 was the latest version that worked without issues in his testing.